### DISCUSSION TITLE
Temperature deltas and heating time for multi-extruders

### DISCUSSION THREAD

**Discussion by 24c** [supermerill/SuperSlicer#907] [Community contribution]
Hi,

I'd like to do away with an ooze/draft shield if possible, because my machine doesn't need it, so would it possible to remove the draft shield by say activating a check box? or possibly allowing for a heating time period by including a temperature heating rate aka ºC/sec placeholder. This way I could do a M104 command for the next extruder, then post processing the GCode to look a few lines back and insert the heating command, so that the filament is at the right temperature to extrude.

I know Cura can do something similar with Standby Temperatures, just wonder if this is being considered in SuperSlicer

Also there is this link in GitHub which is similar too, which does the same pre-heating, but uses a purge/prime tower.

Thanks
Mike

**Comment by supermerill** [comment]
> remove the draft shield

untick `draft shield`

> Standby Temperatures

`multiple extruders` -> `ooze prevention`
